packages/core/src/Table/stories/TableSection/TableEditable.tsx

```diff
@@ -585,7 +585,13 @@ const Table = <T extends Data>({
         </HvTableContainer>
       </LoadingContainer>
       {page.length > 0 ? (
-        <HvPagination {...getHvPaginationProps?.()} />
+        <HvPagination
+          {...getHvPaginationProps?.()}
+          labels={{
+            pageSizePrev: "",
+            pageSizeEntryName: `of ${(data && data.length * pageCount) || 0}`,
```

Comment:
Sometimes the page count is wrong (for example if you go to the last page, the page count is 900)

```suggestion
            pageSizeEntryName: `of ${(totalRecords}`,
```

You can get the `totalRecords` from `useServerData`. You'll  need to pass it to the `Table` component as a prop.

Reply:
Got it, forgot about that. Updated.